iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/tripping/AbstractInjectionTripping.java

```diff
@@ -26,10 +26,10 @@ protected AbstractInjectionTripping(String id) {
     }
 
     @Override
-    public void traverse(Network network, Set<Switch> switchesToOpen, Set<Terminal> terminalsToDisconnect) {
+    public void traverse(Network network, Set<Switch> switchesToOpen, Set<Terminal> terminalsToDisconnect, Set<Terminal> affectedTerminals) {
```

Comment:
"affected" is not the correct word: traversedTerminal?

Reply:
done